src/SwedbankPay.Sdk.Tests/TestBuilders/ConsumersRequestContainerBuilder.cs

```diff
@@ -8,18 +8,16 @@ namespace SwedbankPay.Sdk.Tests.TestBuilders
 {
     public class ConsumersRequestContainerBuilder
     {
-        private EmailAddress email;
         private Language language;
-        private Msisdn msisdn;
         private NationalIdentifier nationalIdentifier;
         private Operation operation;
         private IEnumerable<RegionInfo> shippingAddressRestrictedToCountryCodes;
 
 
         public ConsumersRequest Build()
         {
-            return new ConsumersRequest(this.language, this.shippingAddressRestrictedToCountryCodes, this.operation, this.msisdn,
-                                        this.email, this.nationalIdentifier);
+            return new ConsumersRequest(this.language, this.shippingAddressRestrictedToCountryCodes, this.operation, null,
```

Comment:
Always sending `null` for `msisdn` and `email` seems like an unintentional "fix".

Reply:
I removed them, is this fine?